internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -293,16 +272,37 @@ Resources:
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
 
+{{- if .HTTPSListener}}
+{{- if not .Aliases}}
+  LoadBalancerDNSAlias:
```

Comment:
should this be something like `if not aliases and not importing certs`?

Reply:
Actually, now that we require them to specify alias whenever they use imported certs for the env, we can skip this condition. Otherwise we might put redundant validation here which is potentially confusing.